witnet.toml

```diff
@@ -11,3 +11,6 @@ db_path = ".witnet-rust-testnet-3"
 
 [consensus_constants]
 checkpoint_zero_timestamp = 1559347200
+
+[log]
+level = "debug"
```

Comment:
I think the default should be "info"

Reply:
This is kind of the defaults for development, `bootstrap_peers_period_seconds` is set to `15` for example, the defaults in the crate config use `info`